.github/workflows/release.yml

```diff
@@ -0,0 +1,38 @@
+name: release
+
+on:
+  workflow_dispatch:
+    inputs:
+      semver:
+        description: "The semver to use"
+        required: true
+        default: "patch"
+        type: choice
+        options:
+          - patch
+          - minor
+          - major
+      tag:
+        description: "The npm tag"
+        required: false
+        default: "latest"
+  pull_request:
+    types: [closed]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: pnpm/action-setup@v2
+          with:
+            version: 7
+      - uses: nearform/optic-release-automation-action@v3
+        with:
+          npm-token: ${{ secrets[format('NPM_TOKEN_{0}', github.actor)] || secrets.NPM_TOKEN }}
+          optic-token: ${{ secrets[format('OPTIC_TOKEN_{0}', github.actor)] || secrets.OPTIC_TOKEN }}
```

Comment:
these secrets need to set up in the repo and a decision needs to be made about who can release.

Reply:
The `OPTIC_TOKEN` is not required if I use an `npm` publish token right?

Reply:
correct. the npm automation token allows you to bypass 2FA. if you want to go that way the release process is simpler (though less secure of course). you would still need to decide whether to have a single NPM token or one per person who can release. the second option allows to restrict who can release to only the people who have a NPM_TOKEN_{username} configured in the repo secrets

Reply:
Perfect, I already set up an `NPM_TOKEN` for the next release. Next week we'll catch up to decide who has the rights to make a release, thanks a lot for this PR